onadata/apps/logger/tests/models/test_xform.py

```diff
@@ -140,17 +144,17 @@ def test_get_survey_element(self):
 
         # get fruita element by name
         fruita = xform.get_survey_element("fruita")
-        self.assertEqual(fruita.get_abbreviated_xpath(), "a/fruita")
+        self.assertEqual(get_abbreviated_xpath(fruita.get_xpath()), "a/fruita")
 
         # get exact choices element from choice abbreviated xpath
-        fruita_o = xform.get_survey_element("a/fruita/orange")
-        self.assertEqual(fruita_o.get_abbreviated_xpath(), "a/fruita/orange")
+        # fruita_o = xform.get_survey_element("a/fruita/orange")
```

Comment:
@ukanga Why were these tests commented?

Reply:
@kelvin-muchiri PyXForm choice options no longer refer to the parent; hence, the abbreviated path will simply be the choice name.